packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTClusterManagersController.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+@import GoogleMapsUtils;
+
+NS_ASSUME_NONNULL_BEGIN
+
+// Defines cluster managers controller interface which
+// is responsible for adding/removing/returning one or more cluster managers.
+@interface FLTClusterManagersController : NSObject
+
+/// Initializes FLTClusterManagersController.
+///
+/// @param methodChannel A Flutter method channel used to send events.
+/// @param mapView A map view that will be used to display clustered markers.
+- (instancetype)initWithMethodChannel:(FlutterMethodChannel *)methodChannel
+                              mapView:(GMSMapView *)mapView;
+
+/// Creates ClusterManagers and initializes them.
+///
+/// @param clusterManagersToAdd List of clustermanager object data.
+- (void)addClusterManagers:(NSArray<NSDictionary *> *)clusterManagersToAdd;
+
+/// Removes requested ClusterManagers from the controller.
+///
+/// @param identifiers List of clusterManagerIds to remove.
+
+- (void)removeClusterManagersWithIdentifiers:(NSArray<NSString *> *)identifiers;
+
+/// Returns the ClusterManager for the given identifier.
+///
+/// @param identifier identifier of the ClusterManager.
+/// @return GMUClusterManager if found otherwise nil.
+- (GMUClusterManager *)clusterManagerWithIdentifier:(NSString *)identifier;
```

Comment:
The API declaration asserts that the return value will never be null, but the comment just above says the opposite.

Reply:
added `nullable` to the signature